uitestcore/interrogator.py

```diff
@@ -14,16 +14,18 @@ class Interrogator:
     And get_attribute -> String
     """
 
-    def __init__(self, driver, finder, existing_logger=None):
+    def __init__(self, driver, finder, wait_time, existing_logger=None):
         """
         Default constructor which passes the control of webDriver to the current page
         :param driver: the Selenium web driver
         :param finder: Finder used to find elements before interrogating
+        :param wait_time: number of seconds as an Integer, defaults to 10
```

Comment:
I would remove the "defaults to 10" part of this comment because it could be misleading.